### DISCUSSION TITLE
Mia-Platform Console - Design section - Last used branch selected by default

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#7] [Ideas]
Hi all,
I don't like that selecting the _Design section_ of a project the console opens by default the last used branch.
I think this behavior is dangerous, often I notice that I'm working on the wrong branch after a while.

As suggested by @danibix95 a _User settings_  section into the Mia-Platform console could be very useful for customizations like this one.

**Comment by giulioroggero** [comment]
Good suggestion

**Comment by frengooo** [comment]
@davidebianchi @fredmaggiowski should I add this feature into the product roadmap?

**Reply by fredmaggiowski** [reply]
Yes please

**Reply by frengooo** [reply]
[Task created](https://makeitapp.atlassian.net/browse/MIAROAD-220).
You can close the discussion.

**Reply by fredmaggiowski** [reply]
discussion can't be closed lol
